playbooks/roles/ansible-role-django-ida/templates/defaults/main.yml.j2

```diff
@@ -29,6 +29,7 @@
     PORT: '3306'
     ATOMIC_REQUESTS: true
     CONN_MAX_AGE: 60
+    OPTIONS: '{{ '{{' }} {{ role_name|upper}}_MYSQL_OPTIONS }}'
```

Comment:
@johnbaldwin This is edX's meta-role to generate Ansible code for other IDA roles. This is crazy Jinja escape syntax.. the inner `'{{'` is the thing that gets printed and the surrounding `{{ }}` are the escapers. The syntax in this line is consistent with the syntax elsewhere in this file, so you can trust that it works :)